resqpy/well.py

```diff
@@ -305,25 +305,9 @@ def __init__(self, parent_model, uuid=None, title=None, deviation_survey_root=No
          warnings.warn("Argument deviation_survey_root is deprecated, please use uuid")
          uuid = rqet.uuid_for_part_root(deviation_survey_root)
 
-      if uuid is None:
-         self.uuid = bu.new_uuid()
-      else:
-         self.uuid = uuid
-         self.load_from_xml()
-
-   @property
-   def root(self):
-      """Node corresponding to self.uuid"""
-      if self.uuid is None:
-         raise ValueError('Cannot get root if uuid is None')
-      return self.model.root_for_uuid(self.uuid)
-
-   @property
-   def part(self):
-      """Part corresponding to self.uuid"""
-      if self.uuid is None:
-         raise ValueError('Cannot get part if uuid is None')
-      return self.model.part_for_uuid(self.uuid)
+      super().__init__(
+         model=parent_model, uuid=uuid, title=title, originator=originator
+      )
 
```

Comment:
Don't we want to call load_from_xml() if uuid (or deviation_survey_root) is provided?

Reply:
Should be handled in the base class